projects/storefrontlib/src/cms-components/checkout/components/payment-method/payment-method.component.ts

```diff
@@ -156,11 +156,14 @@ export class PaymentMethodComponent implements OnInit, OnDestroy {
     paymentDetails: PaymentDetails;
     billingAddress: Address;
   }): void {
-    this.getDeliveryAddressSub = this.checkoutDeliveryService
-      .getDeliveryAddress()
-      .subscribe(address => {
-        billingAddress = address;
-      });
+    if (!billingAddress) {
+      this.getDeliveryAddressSub = this.checkoutDeliveryService
+        .getDeliveryAddress()
+        .subscribe(address => {
```

Comment:
If this method called multi times, then only the last subscription will be unsubscribed.

Reply:
Fixed ;)